### ISSUE TITLE
skopeo does not work against docker daemons at version 25.0.0 or higher 

### ISSUE THREAD

**Issue by bainsy88** [containers/skopeo#2202]
This is a result of the following issue in contianers/image - https://github.com/containers/image/issues/2259.  

However I thought it would be worth a separate issue on skopeo so that we can track pulling in a new version of containers image when a fix has been made.

**Comment by mtrmac** [comment]
Thanks! Let’s discuss the details in https://github.com/containers/image/pull/2260 .

**Comment by mtrmac** [comment]
A c/image fix was merged, and it will be included in Skopeo before tagging a new release.

In the meantime, a possible workaround:
```console
# cat /etc/systemd/system/docker.service.d/version.conf 
[Service]
Environment=DOCKER_MIN_API_VERSION=1.22
```

**Comment by TomSweeneyRedHat** [comment]
Just noting that https://github.com/containers/image/pull/2260 has merged.  Once the latest c/image (> v5.29.1) is vendored here, we can close this Issue

**Comment by mtrmac** [comment]
This fix was released in Skopeo 1.14.2.

**Comment by mirekphd** [comment]
> ```
> DOCKER_MIN_API_VERSION
> ```

Here is the complete procedure of reconfiguring the Docker daemon to make it see this env variable:

```
$ sudo mkdir -p /etc/systemd/system/docker.service.d
$ printf "[Service] \n\nEnvironment=DOCKER_MIN_API_VERSION=1.22\n" | sudo tee /etc/systemd/system/docker.service.d/version.conf
$ sudo systemctl daemon-reload && sudo systemctl restart docker

```
